browser/ui/views/toolbar/brave_app_menu.cc

```diff
@@ -30,7 +30,9 @@ BraveAppMenu::~BraveAppMenu() = default;
 
 void BraveAppMenu::RunMenu(views::MenuButtonController* host) {
   AppMenu::RunMenu(host);
-  menu_metrics_->RecordMenuShown();
+  if (menu_metrics_) {
```

Comment:
Hmm, it's strange. this file `brave_app_menu.cc` is not built for android.

Reply:
Ah.. you just gaurded for safe?

Reply:
I think adding `DCHECK(menu_metrics_);` in ctor is more better instead of checking `if (menu_metrics_)` because this file is not included in Android and it should not be `nullptr` always.

Reply:
yes, good point